internal/secrets/gcpkms/example_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcpkms_test
+
+import (
+	"context"
+
+	"github.com/google/go-cloud/internal/secrets/gcpkms"
+)
+
+func ExampleCrypter_Encrypt() {
+	ctx := context.Background()
```

Comment:
For here and the other example: it's helpful to over-explain each step in examples so that the reader knows what's going and on, and especially what they need to change.

Reply:
Added some comments, thanks!